build/media-suite_helper.sh

```diff
@@ -661,15 +661,8 @@ do_pkgConfig() {
 do_readoptionsfile() {
     local filename="$1"
     if [[ -f "$filename" ]]; then
-        < $filename dos2unix |
```

Comment:
Was this just removing the comments? If so, revert it. The comments are useful as documentation for how to format the options files.

Reply:
It was mainly for removing the dos2unix. Reverted